nav2_minimal_tb4_sim/launch/spawn_tb4.launch.py

```diff
@@ -14,22 +14,20 @@
 # limitations under the License.
 
 import os
-from pathlib import Path
 
 from ament_index_python.packages import get_package_share_directory
 
 from launch import LaunchDescription
-from launch.actions import AppendEnvironmentVariable
 from launch.actions import DeclareLaunchArgument
 from launch.conditions import IfCondition
 from launch.substitutions import LaunchConfiguration
 
-from launch_ros.actions import Node
+from launch_ros.actions import Node, PushRosNamespace
 
 
 def generate_launch_description():
     sim_dir = get_package_share_directory('nav2_minimal_tb4_sim')
-    desc_dir = get_package_share_directory('nav2_minimal_tb4_description')
+    # desc_dir = get_package_share_directory('nav2_minimal_tb4_description')
```

Comment:
Just remove the line if its not needed

Reply:
The rest of its usage is under commented code that imho should be addressed at some point (spawning the robot through SDF the same way it's done for TB3 i.e. [here](https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation/pull/15/files/d3aaa690d4c531d5eca1247dfbf617235dd7663d#diff-34b6904977e9bf9fb33a4a8f9ef6a02d757012718d503bfc3ca01e6e4f67db6eL68-L71). I commented it as consistency with the rest of the TODO and to make it easier to address it later, otherwise might as well remove all the related commented code.

Reply:
I agree, remove all commented out code

Reply:
Done https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation/pull/15/commits/cdacf9237c2f80502a5a3fa1deb9061c8643c4b4